v2.0/backup.md

```diff
@@ -26,9 +26,10 @@ Dependent objects should be backed up at the same time as the objects they depen
 
 Object | Depends On
 -------|-----------
-Table with [foreign key](foreign-key.html) constraints | The table it `REFERENCES` (however, this dependency can be [removed during the restore](restore.html#skip_missing_foreign_keys))
-[Views](views.html) | The tables used in the view's `SELECT` statement
-[Interleaved tables](interleave-in-parent.html) | The parent table in the [interleaved hierarchy](interleave-in-parent.html#interleaved-hierarchy)
+Table with [foreign key](foreign-key.html) constraints | The table it `REFERENCES` (however, this dependency can be [removed during the restore](restore.html#skip_missing_foreign_keys)).
+Table with a [sequence](create-sequence.html) | <span class="version-tag">New in v2.0:</span> The sequence it uses (however, this dependency can be [removed during the restore](restore.html#skip_missing_sequences-new-in-v2-0).
```

Comment:
nit: In this and the preceding row, could you change the parenthetical to a stand-alone sentence (or separate with a semi-colon)?